### PR TITLE
Compositor::blendFrames replaces assembleFramesSorted

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: eq/compositor.h [hunk 1]
```diff
@@ -86,10 +86,9 @@ class EQ_API Compositor
      * @return the number of different subpixel steps assembled.
      * @version 1.0
```
Comment (tribal-tec) [inline eq/compositor.h]: version 1.0? API breakage?

Comment (eile) [inline eq/compositor.h]: Yes.

**Comment by eile** [comment]
ping - bumped version to 1.11

File: eq/compositor.h [hunk 2]
```diff
@@ -84,12 +84,10 @@ class EQ_API Compositor
      * @param blendAlpha blend color-only images if they have an alpha
```
Comment (tribal-tec) [inline eq/compositor.h]: rm doxygen blendAlpha

**Comment by tribal-tec** [comment]
+1 otherwise

File: eq/compositor.h (hunk 1 above)
Comment (bilgili) [inline eq/compositor.h]: There is no blendAlpha parameter in doc

Comment (bilgili) [inline eq/compositor.h]: All examples have accum as 0. May it be default parameter ?

**Pull request merged**